cesium/channel_test.go

```diff
@@ -74,6 +75,162 @@ var _ = Describe("Channel", func() {
 					),
 				)
 			})
+
+			Describe("Rekey", func() {
```

Comment:
These cases look good, but I think they miss the edge case mentioned above, where we don't actually change the key of the channel in the meta file.